packages/js/components/src/experimental-tree-control/tree.tsx

```diff
@@ -20,24 +23,60 @@ export const Tree = forwardRef( function ForwardedTree(
 		ref,
 	} );
 
-	if ( ! items.length ) return null;
+	const createValueExists =
+		items.findIndex( ( i ) => i.data.name === props.createValue ) !== -1;
+
 	return (
-		<ol
-			{ ...treeProps }
-			className={ classNames(
-				treeProps.className,
-				'experimental-woocommerce-tree',
-				`experimental-woocommerce-tree--level-${ level }`
+		<>
+			<ol
+				{ ...treeProps }
+				className={ classNames(
+					treeProps.className,
+					'experimental-woocommerce-tree',
+					`experimental-woocommerce-tree--level-${ level }`
+				) }
+			>
+				{ items.map( ( child, index ) => (
+					<TreeItem
+						{ ...treeItemProps }
+						isExpanded={ props.isExpanded }
+						key={ child.data.id }
+						item={ child }
+						index={ index }
+					/>
+				) ) }
+			</ol>
+			{ props.allowCreate && ! createValueExists && (
```

Comment:
Given this is not part of the `TreeItem` data it doesn't allow a user to select it using the keyboard, would this still be possible outside of the `TreeItem` component?

Reply:
I will take a look. Maybe I could use a "fixed Tree Item" for this functionality